templates/vue/src/components/modals/NodeModal/forms/ContentForm/CaptionRow.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <b-card
+    :bg-variant="isPending ? 'danger' : 'secondary'"
+    text-variant="light"
+    class="mt-2"
+    data-qa="caption-row"
+  >
+    <b-row align-v="center" class="mb-2 mx-0">
+      <div :id="`${captionContainerId}-toggle`" v-b-toggle="captionContainerId">
+        <b>{{ title }}</b>
+        <span v-if="isDefault" class="ml-2">
+          (default)
+        </span>
+        <span v-if="isPending && errorMessage" class="ml-2">
+          <i
+            :id="`caption-error-message-${caption.id}`"
+            class="far fa-question-circle"
+            tabindex="0"
+          ></i>
+          <b-tooltip :target="`caption-error-message-${caption.id}`">
+            This caption could not be uploaded. {{ errorMessage }}
+          </b-tooltip>
+        </span>
+      </div>
+      <div class="ml-auto d-flex align-items-center">
+        <b-button
+          class="mr-1"
+          size="sm"
+          :variant="isRemovable ? 'danger' : 'light'"
+          :disabled="!isRemovable"
```

Comment:
I think it was just a choice; when the user turns captions "On", they must provide at least one caption. If it would be clearer, I can change it so that deleting the last caption turns off the caption toggle.

Reply:
I think that makes more sense. Also since the toggle's state is related to the number of captions, it shouldn't require more code for it to work.